supabase/functions/wallets-list/index.ts

```
@@ -210,10 +210,24 @@ serve(async (req) => {
+    const { data: userProfile, error: profileError } = await supabase
+      .from('users')
+      .select('plan')
+      .eq('user_id', userId)
+      .single()
+
+    if (profileError) {
+      console.error('Failed to fetch user profile:', profileError)
+      // Default to free tier if profile fetch fails
+      var plan = 'free'
+    } else {
+      var plan = userProfile?.plan || 'free'
```

Biome (2.1.2)

[error] Shouldn't redeclare 'plan'. Consider to delete it or rename it.

'plan' is defined here:


(lint/suspicious/noRedeclare)